src/components/Configure/actions/read/onSaveReadCreateInstallation.ts

```diff
@@ -76,6 +76,12 @@ const generateCreateReadConfigFromConfigureState = (
     },
   };
 
+  // insert proxy into config if it is enabled
+  const isProxyEnabled = hydratedRevision.content.proxy?.enabled;
```

Comment:
Can you put this line into a function that takes in a hydratedRevision and returns a boolean indicating whether or not proxy is enabled (we might change the format of the hydratedRevision in the future where if it has any other keys then `enabled: true` is not explicitly required). So if it was abstracted into a helper function then that logic is easy to change in the future.